src/views/Main/index.jsx

```diff
@@ -297,12 +289,6 @@ class MainContainer extends Component {
             </Suspense>
             {doneLoading === false && <Spinner loading /> }
           </div>
-          <BottomNavigation
```

Comment:
why are you removing this?

Reply:
Did not intend to remove this, adding it back.